lib/tasks/screens.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Mix.Tasks.Addict.Gen.Screens do
+  use Mix.Task
+  import Mix.Generator
+  alias Addict.Utils
+
+  @shortdoc "Generate templates and view for Addict"
+  @moduledoc """
+  Generates an Addict view and templates
+      mix addict.gen.screens
+  """
+  def run(opts) do
+    bindings = [
+      module: Module.concat(Utils.app_module, AddictView),
+      view: Module.concat(Utils.app_module, View)
+    ]
+
+    view_file = Path.join(["web", "views", "addict_view.ex"])
+                |> Path.relative_to(Mix.Project.app_path)
+    create_file view_file, view_template(bindings)
```

Comment:
Hey @trestrantham, I was looking at the way this is implemented in detail and I don't quite get where this `view_template` comes from?